### ISSUE TITLE
Method get_pixmap() make the program exit without any exceptions or messages

### ISSUE THREAD

**Issue by Jianxinzhu-pact** [pymupdf/PyMuPDF#3727]
### Description of the bug

When using page.get_pixmap() method, the program simply exits without any prompts(Both in Windows and Ubuntu) and cannot catch the exception.

### How to reproduce the bug

Sample code:
```python
import fitz
# can download the pdf from http://www.christtherock.org/wp-content/uploads/2020/03/7-Basic-Commands-Week-6.pdf
pdf_path = r"E:\Files\general-pdf\pdf\us\c63a\13e5\c63a13e590b5ec65994f84e179925e49.pdf"
doc = fitz.open(pdf_path)
for page in doc:
    try:
        page.get_pixmap(matrix = fitz.Matrix(2,2))
    except:
        print("error")
```
When using vscode debug the code, it exits when call `_mupdf.fz_run_display_list`.
When using the jupyter notebook, the kernal was crashed with the exitCode 3221225477.
In Ubuntu, it exit with a message `Segmentation fault`

### PyMuPDF version

1.24.9

### Operating system

Windows

### Python version

3.9

**Comment by Jianxinzhu-pact** [comment]
You could find the url in the sample code comment. 
`http://www.christtherock.org/wp-content/uploads/2020/03/7-Basic-Commands-Week-6.pdf`

**Comment by JorjMcKie** [comment]
This is an upstream bug (MuPDF). I will transfer this bug to their issue management.

MuPDF issue link: https://bugs.ghostscript.com/show_bug.cgi?id=707925

**Comment by Jianxinzhu-pact** [comment]
How can i update the pyMuPDF to fix this problem?

**Comment by Jianxinzhu-pact** [comment]
> How can i update the pyMuPDF to fix this problem?

Have built the pyMuPDF in local and process the PDF successful. Thanks.

**Comment by JorjMcKie** [comment]
Ah, I am glad to see that the solution does work for you. You obviously generated PyMuPDF using the MuPDF version from its GitHub repository?

**Comment by Jianxinzhu-pact** [comment]
> Ah, I am glad to see that the solution does work for you. You obviously generated PyMuPDF using the MuPDF version from its GitHub repository?

Yes. From this repo: https://github.com/ArtifexSoftware/mupdf

**Comment by julian-smith-artifex-com** [comment]
Reopening until we've made a release that includes the fix.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.24.10.
